### ISSUE TITLE
Feature request: Add a browse button in the View Options -> Log File tab

### ISSUE THREAD

**Issue by GoogleCodeExporter** [snakefoot/snaketail-net#8]
```
SnakeTail is a great program and especially the keyword highlight and custom 
colors are a great way to keep track of log file.

I monitor log files that frequently have a different file name. For example, 
there are log files that each day write the data to a new file. Now, with the 
current version of SnakeTail 1.5.1, I have two options to manage this:

1). I can choose File -> Open File, and then browse to the correct log file. 
This works simple and makes it easy to find the most recent log file to 
monitor. However, this means that the formatting and keyword highlight have to 
be manually configured again.

2). I can right click in a already open log file, choose View Options -> then 
Log File, and then change the log location in the File Path box. This will 
enable me to keep the formatting and keyword highlight, however I've to 
manually type the file path there. In effect that means I have to browse to the 
directory, copy this on into the File Path box. Then search the correct log 
file in the directory, press F2 to select the file name, and copy also this to 
the File Path box. This is, like option 1, also a tedious process for multiple 
log files.

To summarize: 
It would be great if a 'browse for'-button (perhaps something small like three 
dots) could be made in the View Options -> Log File so that the user easily can 
browse to his new log file while keeping the formatting and keyword highlight 
settings.
```

Original issue reported on code.google.com by `josjuram...@gmail.com` on 1 Sep 2011 at 6:35
